resources/views/hr/application/rejection-modal.blade.php

```diff
@@ -41,6 +41,28 @@
                     <span aria-hidden="true">&times;</span>
                 </button>
             </div>
+                <div class="form-check">
+                    <form action="#" class="textbox">
+                        <h5><b> Select Reasons: </b></h5>
+                        <input type="checkbox" id="rejectReason1"> No responce <br>
```

Comment:
Whenever you are using a text inside a blade tag, always sanitize it using `{{ __('') }}`

```suggestion
                        <input type="checkbox" id="rejectReason1"> {{ __('No response') }} <br>
```

Reply:
Sure let me change it  for all